lib/mayaUsd/ufe/UsdContextOps.cpp

```diff
@@ -619,17 +619,17 @@ Ufe::ContextOps::Items UsdContextOps::getItems(const Ufe::ContextOps::ItemPath&
         items.emplace_back(kUSDLayerEditorItem, kUSDLayerEditorLabel, kUSDLayerEditorImage);
 #endif
 
-        // Top-level items (do not add for gateway type node):
-        if (!fIsAGatewayType) {
 #ifdef UFE_V3_FEATURES_AVAILABLE
-            if (PrimUpdaterManager::getInstance().canEditAsMaya(path())) {
-                items.emplace_back(kEditAsMayaItem, kEditAsMayaLabel, kEditAsMayaImage);
-                items.emplace_back(kDuplicateAsMayaItem, kDuplicateAsMayaLabel);
-                items.emplace_back(kAddMayaReferenceItem, kAddMayaReferenceLabel);
-                items.emplace_back(Ufe::ContextItem::kSeparator);
-            }
+        if (!fIsAGatewayType &&
+            PrimUpdaterManager::getInstance().canEditAsMaya(path())) {
+            items.emplace_back(kEditAsMayaItem, kEditAsMayaLabel, kEditAsMayaImage);
+            items.emplace_back(kDuplicateAsMayaItem, kDuplicateAsMayaLabel);
+        }
+        items.emplace_back(kAddMayaReferenceItem, kAddMayaReferenceLabel);
```

Comment:
Adding a Maya reference prim can be done both for the gateway node (i.e. proxy shape) and for its children.